ethereum-rpc/src/constructors/express/routes/api/v1/uniswap.js

```diff
@@ -0,0 +1,512 @@
+const router = require("express").Router();
+const ethers = require('ethers');
+const { ChainId, WETH, Fetcher, Trade, Route, TokenAmount, TradeType, Percent } = require('@uniswap/sdk')
+const { query, param, validationResult } = require("express-validator");
+
+
+module.exports = ({
+  models,
+  ethereum: { web3, buildContract, compiler, abi },
+  ...context
+}) => {
+
+  router.post(
+    "/:address/swap/eth", [
+    param("address").trim().isString(),
+    query("tokenFrom").trim().isIn(["eth"]),
+    query("tokenTo").trim().isIn(["dai", "mkr", "uni"]),
```

Comment:
We should force these to either uppercase or lowercase (standard is typically uppercase for these)

Reply:
done